main/reference/zoe-api/zoe-data-types.md

```diff
@@ -83,11 +54,34 @@ is exclusively yours and no one else has access to it.
 
 ## Keyword
 
-An ASCII identifier string that must begin with an upper case letter
-in order to avoid collisions with JavaScript properties such as `toString`.
+A **Keyword** is a unique identifier string within a contract for tying together its
+**proposals**, **payments**, and **payouts**.
+It must be a valid ASCII [identifier](https://developer.mozilla.org/en-US/docs/Glossary/Identifier)
+and start with an upper case letter in order to avoid collisions with JavaScript properties
+such as `toString` when used as a property name in a record.
 (For more detail, see [Why do Zoe keywords have to start with a capital letter? #8241](https://github.com/Agoric/agoric-sdk/discussions/8241).)
 `NaN` and `Infinity` are also not allowed as keywords.
 
+<a name="amountkeywordrecord"></a>
+## KeywordRecord
+
+A **KeywordRecord** is a [CopyRecord](/glossary/#passable) in which every property name
+is a **[Keyword](#keyword)**, such as `harden({ Currency: quatloosBrand, CurrencyUnit: 100n })`.
+Subtypes further constrain property values (for example, an
+**AmountKeywordRecord** is a **KeywordRecord** in which every value is an
+**[Amount](/reference/ertp-api/ertp-data-types.md#amount)** and a
+**PaymentKeywordRecord** is a **KeywordRecord** in which every value is a
+**[Payment](/reference/ertp-api/payment.md)**).
```

Comment:
I know `PaymentPKeywordRecord` is in our API; that is: `Record<Keyword, Promise<Payment>>` that's returned from `getPayouts()`. I don't know of any use of `Record<Keyword, Payment>`.

Reply:
The payments in `E(zoe).offer()` are a `PaymentKeywordRecord`.

Reply:
That should be fixed, but I don't have the necessary context. Would either of you be willing to submit a followup or commit in this PR?

Reply:
A single letter would be a big improvement: change `PaymentKeywordRecord` to `PaymentPKeywordRecord`.

It would be more precise to say "in which every value is a Payment or a Promise for a Payment" but I think it's OK to leave the Promise stuff implicit in this discussion.

Reply:
Done.